lib/node_modules/@stdlib/repl/lib/tokenizer.js

```diff
@@ -34,10 +34,12 @@ var commands = require( './commands.js' );
 // VARIABLES //
 
 var COMMANDS = commands();
+var isUnrecognizedKeyword = contains.factory( [ 'async', 'let' ] );
```

Comment:
You can save some characters by modifying L28 to

```
var contains = require( '@stdlib/array/base/assert/contains' ).factory;
```

and then the lines below become

```
var isUnrecognizedControlKeyword = contains( [ 'await' ] );
```

Reply:
I am also using `contains` for other purposes below.

Reply:
`contains` is also being directly used below.